### DISCUSSION TITLE
EL1 MyOpenMath

### DISCUSSION THREAD

**Discussion by kathypinzon** [TeamBasedInquiryLearning/precalculus#376] [General]
Reviewer: @AbbyANoble 

1-4 - evaluate exponentials (including e)
5-7 - growth or decay
8 - tables linear/exponential/neither
9-12 - write exponential equation given properties

**Comment by AbbyANoble** [comment]
Number 10 asks for the function in the question, but the answer is just the value of a.

**Reply by AbbyANoble** [reply]
Similar in 11.

**Reply by AbbyANoble** [reply]
Number 12 is more clear.
